dashboard/app/controllers/api/v1/pd/application/principal_approval_applications_controller.rb

```diff
@@ -15,14 +15,18 @@ def on_successful_create
       teacher_application = Pd::Application::Teacher1819Application.find_by!(application_guid: @application.application_guid)
       principal_response = @application.sanitize_form_data_hash
 
+      response = principal_response.values_at(:replace_course, :replace_course_other).compact.join(": ")
+      replaced_courses = principal_response.values_at(:replace_which_course_csp, :replace_which_course_csd).compact.join(', ')
+      replace_course_string = "#{response}#{replaced_courses.present? ? ': ' + replaced_courses : ''}".gsub('::', ':')
```

Comment:
super-minor nit: might be worth dropping in a comment about the gsub and why it's useful